### PR TITLE
[android] generate expo package list

### PR DESCRIPTION
**Pull request by wkozyra95**

### CODE REVIEW

File: android/app/src/main/java/host/exp/exponent/generated/BasePackageList.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import java.util.List;
+import org.unimodules.core.interfaces.Package;
+
+public class BasePackageList {
+  public List<Package> getPackageList() {
+    return Arrays.<Package>asList(
+        new expo.modules.ads.facebook.AdsFacebookPackage()
+    );
+  }
+}
```
Comment (tsapeta) [inline android/app/src/main/java/host/exp/exponent/generated/BasePackageList.java]: Shouldn't it be gitignored?

File: android/expoview/build.gradle [hunk 2]
```diff
@@ -113,6 +113,7 @@ configurations.all {
 // WHEN_VERSIONING_REMOVE_FROM_HERE
 // Import gradle helpers for unimodules.
 apply from: "../../node_modules/react-native-unimodules/gradle.groovy"
+apply from: "unimodules.gradle.kts"
```
Comment (tsapeta) [inline android/expoview/build.gradle]: Can we move this `unimodules.gradle.kts` into `gradle.kts` in `react-native-unimodules`?

File: android/expoview/build.gradle [hunk 3]
```diff
@@ -322,4 +323,5 @@ dependencies {
   compileOnly 'expolib_v1.com.squareup.okhttp3:okhttp-urlconnection:3.6.0'
   compileOnly 'expolib_v1.com.squareup.okio:okio:1.13.0'
   // WHEN_PREPARING_SHELL_REMOVE_TO_HERE
+  compile 'com.squareup:javapoet:1.11.1'
```
Comment (tsapeta) [inline android/expoview/build.gradle]: unused?

File: packages/expo-ads-facebook/unimodule.json [hunk 4]
```diff
@@ -1,4 +1,5 @@
 {
   "name": "expo-ads-facebook",
-  "platforms": ["ios", "android"]
+  "platforms": ["ios", "android"],
+  "androidBasePackage": "expo.modules.ads.facebook.AdsFacebookPackage"
```
Comment (tsapeta) [inline packages/expo-ads-facebook/unimodule.json]: Let's make it a list of packages and rename to `androidPackages`

File: android/expoview/build.gradle (hunk 2 above)
Comment (wkozyra95) [reply android/expoview/build.gradle]: kotlin gradle script seems to require naming in form *.gradle.kts

**Pull request closed without merging**